smoketest/scripts/cli/test_protocols_bgp.py

```diff
@@ -221,12 +221,13 @@ def test_bgp_01_simple(self):
 
         self.session.set(base_path + ['parameters', 'router-id', router_id])
         self.session.set(base_path + ['parameters', 'log-neighbor-changes'])
-        # Default local preference (higher=more preferred)
+        # Default local preference (higher = more preferred, default value is 100)
         self.session.set(base_path + ['parameters', 'default', 'local-pref', local_pref])
         # Deactivate IPv4 unicast for a peer by default
         self.session.set(base_path + ['parameters', 'default', 'no-ipv4-unicast'])
         self.session.set(base_path + ['parameters', 'graceful-restart', 'stalepath-time', stalepath_time])
         self.session.set(base_path + ['parameters', 'graceful-shutdown'])
+        self.session.set(base_path + ['parameters', 'ebgp-requires-policy'])
```

Comment:
Please also add a `self.asser()` statement which verifys the proper configuration of this node (how should the FRR config look like?).

Reply:
Ahh sure, yeah I completely went right over that even though I saw it. My bad. I'll add that. Thank you.